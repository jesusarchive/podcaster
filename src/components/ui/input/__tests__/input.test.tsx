import { render } from '@testing-library/react';
import React from 'react';

import Input from '../input';

describe('Input', () => {
  it('should render the input component', () => {
    const { container } = render(<Input />);

    expect(container).toMatchSnapshot();
  });

  it('should render the input component with props', () => {
    const { container } = render(<Input placeholder="input" type="text" value="input" />);

    expect(container).toMatchSnapshot();
  });
});
