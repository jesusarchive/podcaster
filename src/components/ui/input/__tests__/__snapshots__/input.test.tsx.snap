// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render the input component 1`] = `
<div>
  <input
    class="input input-bordered"
  />
</div>
`;

exports[`Input should render the input component with props 1`] = `
<div>
  <input
    class="input input-bordered"
    placeholder="input"
    type="text"
    value="input"
  />
</div>
`;
